### DISCUSSION TITLE
How to add different databases support

### DISCUSSION THREAD

**Discussion by lloveoov** [vendurehq/vendure#1264] [Q&A]
I see Vendure supports multiple databases from developer guide, but I cannot find more inforamtion:
I want to contribute for testing different databases ,and add them to vendure/create tools, thx!

**Comment by michaelbromley** [comment]
Hi,
What databases do you want it to support? Right now we officially support Mysql, Postgres & SQLite, plus API-compatible variants (such as Amazon Aurora etc.)

**Reply by lloveoov** [reply]
Hi,

I see this blob https://www.vendure.io/blog/2021/11/serverless-e-commerce-vendure-on-google-cloud-run/, which host Vendure on Google Cloud Run. 
So I want to check if it is possible host in Microsoft Azure or Amazon.

For example,host vendure in Azure Container App, and connect to Aure SQl Database(same database as Sql Server but host in cloud).


Thanks!

**Reply by michaelbromley** [reply]
Yes, you can host it on Azure or Amazon services. You just need to make sure to use a DB that is compatible with MySQL or Postgres. For example, Amazon has Aurora which I believe comes in both varieties. I would expect Azure to offer both too.

It will not work with SQL Server, as that is not supported by Vendure.

**Reply by lloveoov** [reply]
Thank you!
